targets/CMakeLists.txt

```diff
@@ -9,4 +9,6 @@ elseif("NORDIC" IN_LIST MBED_TARGET_LABELS)
     add_subdirectory(TARGET_NORDIC)     
 elseif("STM" IN_LIST MBED_TARGET_LABELS)
     add_subdirectory(TARGET_STM)    
+elseif("WICED" IN_LIST MBED_TARGET_LABELS)
```

Comment:
`WICED` isn't really a target. 
I'm wondering if this could be an opportunity to move the code/library somewhere else.
Either under `EMW3166` or Connectivity folder.

Reply:
This clean up can happen when we stop supporting old build tools.. It would be great but if we move it and remove `TARGET_` it would become breaking change.
Or can we do it non breaking?

Reply:
I think in order to avoid breaking changes, we'll have to keep the `TARGET_WICED` name.
One option is to put it under `TARGET_STM/TARGET_STM32F4/TARGET_STM32F412xG/TARGET_WIO_EMW3166`
The `wiced_interface` could be put under `Connectivity`.
Anyway, if this is too much of a hassle, can be done in the future.

Reply:
Yes definitely move it when possible. Do the new tools provide for the concept of an external device driver? Or 'host control interface'.

Reply:
If there is external driver, it would be a library that an app/another lib can link to via `target_link_libraries()` , first needs t obe added to CMake via `add_subdirectory()` to make it visible. Does that answer the question?